R/helpers_design.R

```diff
@@ -166,3 +166,24 @@ h_add_dlts <- function(data,
   }
   update(data, x = dose, y = dlts)
 }
+
+
+#' Helper Function to call truth calculation
+#'
+#' @param dose (`number`)\cr current dose.
+#' @param truth (`function`)\cr defines the true probability for a DLT at a dose.
+#' @param this_args (`data.frame`)\cr list of arguments for the truth.
+#' @return The updated `this_truth`.
```

Comment:
True probability of DLT.

Reply:
@robertadamsbayer just to clarify: this comment referred to the return value

Reply:
Thanks @danielinteractive and sorry! Embarrassing! :D